src/add-rmv-edit.js

```diff
@@ -0,0 +1,68 @@
+const todoList = document.getElementById('todo-list');
+const form = document.getElementById('form');
+const taskIn = document.getElementById('taskIn');
+
+const displayTask = () => {
+  const tasks = JSON.parse(localStorage.getItem('listItem')) || [];
+  if (tasks === null) return;
+
+  const sortedList = tasks.slice().sort((a, b) => a.index - b.index);
+  todoList.innerHTML = '';
+  sortedList.forEach((task) => {
+    const list = `
+        <div class="task task-${task.index}">
+            <input type="checkbox" data-btn="${task.index}">
+            <input type="text" class="list" value="${task.description}" data-desc="${task.index}">
+            <button class="move" data-remove="${task.index}"></button>
+        </div>
+    `;
+    todoList.insertAdjacentHTML('beforeend', list);
+  });
+};
+
+const saveLocalStorage = () => {
+  const tasks = JSON.parse(localStorage.getItem('listItem')) || [];
+  const tasksItem = taskIn.value;
+  taskIn.value = '';
+  if (tasksItem === null) return;
+  const task = {
+    description: tasksItem,
+    completed: false,
+    index: tasks.length,
+  };
+  const filtered = [...tasks, task];
+  localStorage.setItem('listItem', JSON.stringify(filtered));
+};
```

Comment:
As mentioned in the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Implement%20a%20function%20for%20adding%20a%20new%20task%20(add%20a%20new%20element%20to%20the%20array).), adding an item should be in its own function, the item should be added to an array that's declared (check the previous comment) and it should be separate from saving to the local storage. After an item is added to that array, then you can go ahead and save that array to the local storage.